### ISSUE TITLE
handler.js

### ISSUE THREAD

**Issue by Whooolia** [fac-15/schmidt-resources-ltd#18]
I am not sure, but line 28-30 doesn't look like necessary(?) I don't find it has been using on the website or html file.

**Comment by boi87** [comment]
True! We've put it there just because we might have needed them in the future. We have been ambitious!
